spec/samples/IVC_color-bezkrovny.json

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
Good call on tracking Bezkrovny's values as well for the tests.

In general, I'm wondering what the minimum relevant resolution is for this. I would like to consolidate results if we can (even if it's at the expense of locking down the values less or just overall tracking correlation with subjective scores stays relevant). So it would be great if we could have a single file (IVC values) and some tolerance range.

I took a look at comparing the difference between Weber and Bezkrovny's to the original IVC values:

![image](https://user-images.githubusercontent.com/3877773/90338305-a045a600-df9d-11ea-95af-9818ab6818a0.png)

It looks like Weber deviates ~30% more from IVC values than Bezkrovny's (0.0155 vs 0.0202) and tends to always be a slightly more aggressive degradation value (most difference is negative) but both differences are rather small.

From a user perspective, when you try to see what the degradation is, e.g. `0.885±0.02`, that ±2% for using Weber vs fast is unlikely to make a difference.

What's more, looking at the correlation with subjective scores, both Bezkrovny's and Weber's correlate slightly better with subjective values than the original IVC ones:


Original | Weber | Bezkrovny
-- | -- | --
76.18% | 77.72% | 77.77%

Reply:
In terms of code changes, how would you feel about keeping a single IVC file but adding a tolerance in the test results for `±0.08`? Currently the max difference in Bezkrovny is 0.054 and 0.072 for Weber.

Later on (not on this PR) we can add a check to make sure that the overall correlation with mean objective scores doesn't worsen

Reply:
I really would like to keep values per image because I want to keep optimizing the algorithm and making sure the output stays the same.  I have no objection to adding a tolerance check.

-- Where did you get the subjective score values to look at?  I'm interested in those.

Reply:
The results of the subjective quality assessment are available at https://github.com/obartra/ssim/blob/master/spec/samples/IVC_SubQualityDB/MOS.xls I've put them on a spreadsheet here https://docs.google.com/spreadsheets/d/1RzRq5oCqafl5KSpDgmYf-rjaX-dJRCh6p8IAA0oM4DY/edit?usp=sharing

Reply:
Also re keeping the per test results to avoid changes that's fine

Reply:
How does the Correl function work in Google Sheets.  Are you comparing MOS/5 against the SSIM values?

Reply:
Yeah pearson's correlation between MOS and SSIM values